modules/bbmap/align/main.nf

```diff
@@ -0,0 +1,59 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process BBMAP_ALIGN {
+    tag "$meta.id"
+    label 'process_high'
```

Comment:
Does bbmap really require so much memory? 72G just for mapping seems quite a lot...?

Reply:
I suppose not in all cases. For my use case it will need all the memory one has, but I'm happy to set this lower here and leave that to the pipeline code. I'll move it to `process_medium`.